src/Application/Service/Async/ResourceInvalidationSubscriber.php

```
@@ -158,10 +214,253 @@ public function run(): void
+    public function markHealthy(): void
+    {
+        if ($this->stopping) {
+            return;
         }
+
+        $recoveredFrom = $this->alarm->recordDwell(ReconnectAlarmPolicy::HEALTHY_DWELL_SECONDS);
+        if ($recoveredFrom === null) {
+            return;
+        }
+
+        StaticLoggerBridge::info('ssr', 'Resource-invalidation subscribe loop recovered', [
```

PHPStan (2.2.8)

[error] Call to an undefined static method Semitexa\Core\Log\StaticLoggerBridge::info().


(staticMethod.notFound)